packages/odyssey-react-mui/src/FieldError.tsx

```diff
@@ -36,7 +35,13 @@ const FieldError = ({
   const { t } = useTranslation();
 
   return (
-    <FormHelperText data-se={testId} error id={id} translate={translate}>
+    <FormHelperText
+      component="div"
```

Comment:
Had to change to `div` to fix invalid nesting of `div` inside `p`

Reply:
Good catch 👍